content/en/docs/tasks/security/authz-http/index.md

```diff
@@ -32,178 +30,259 @@ the product page, you can see the following sections:
+You will start with a simple `deny-all` policy that rejects all requests to the workload, and then
+grant the permission gradually and incrementally to allow more access to the workload.
+
+1. Apply a default `deny-all` policy for workloads in the default namespace
+
+    The following command creates a `deny-all` policy with no workload selector, the policy will
+    select all workloads in the default namespace. The policy will deny all access and we will
+    add more policies to grant the access incrementally in following steps.
+
+    {{< text bash >}}
+    $ kubectl apply -f - <<EOF
+    apiVersion: security.istio.io/v1beta1
+    kind: AuthorizationPolicy
+    metadata:
+      name: deny-all
+    spec:
+      {}
+    EOF
+    {{< /text >}}
+
+    Point your browser at the Bookinfo `productpage` (`http://$GATEWAY_URL/productpage`).
+    Now you should see `"RBAC: access denied"`. The error shows that the configured `deny-all` policy
+    is working as intended, and Istio doesn't have any rules that allow any access
+    to workloads in the default namespace.
+
+1. Allow access to the `productpage` workload
+
+    The following command creates a `productpage-viewer` policy which allows
+    access to the `productpage` workload with `GET` method for all users and workloads.
+
+    {{< text bash >}}
+    $ kubectl apply -f - <<EOF
+    apiVersion: "security.istio.io/v1beta1"
+    kind: "AuthorizationPolicy"
+    metadata:
+      name: "productpage-viewer"
+      namespace: default
+    spec:
+      selector:
+        matchLabels:
+          app: productpage
+      rules:
+      - to:
+        - operation:
+            methods: ["GET"]
+    EOF
+    {{< /text >}}
+
+    Point your browser at the Bookinfo `productpage` (`http://$GATEWAY_URL/productpage`).
+    Now you should see the "Bookinfo Sample" page. But there are errors `Error fetching product details`
+    and `Error fetching product reviews` on the page. These errors are expected because
+    we have not granted the `productpage` workload access to the `details` and `reviews` workloads.
+    We will fix the errors in the following steps.
+
+1. Allow access to the `details` and `reviews` workloads
+
+    The following command has the following effects:
+
+    * Creates a `details-viewer` policy which allows access with `GET` method to the `details` workload for
+      service account `cluster.local/ns/default/sa/bookinfo-productpage` (representing the `details` workload).
+
+    * Creates a `reviews-viewer` policy which allows access with `GET` method to the `reviews` workload for
+      service account `cluster.local/ns/default/sa/bookinfo-productpage` (representing the `reviews` workload).
+
+    Note that in the [setup step](#before-you-begin), we created the `bookinfo-productpage` service account
+    for the `productpage` workload. This `bookinfo-productpage` service account is the authenticated identify
```

Comment:
There appears to be a typo but I ask, do the reader need to know this to complete the task? If your goal is for readers to make the connection between the SA and the value in `-from: -source: principals:` in the YAML file, then use the full value. Don't assume readers know how to complete the path with just the SA name. Also, you would need to explicitly call out and explain the fields involved or at the very least point readers towards a page with the explanation.
IMO, you don't need to go beyond my suggestion below though.
```suggestion
    for the `productpage` workload. Once you apply the configuration, only the `cluster.local/ns/default/sa/bookinfo-productpage` service account is authorized to access the `details` and `reviews` workloads through `GET` methods.
```

Reply:
Reworded to simplify, we don't need to refer to the Bookinfo page at all.